Dockerfile

```diff
@@ -32,4 +34,4 @@ EXPOSE 36307
 
 VOLUME /opt/sphinx/conf
 
-CMD searchd --nodetach --config /opt/sphinx/conf/sphinx.conf
+CMD searchd --nodetach --config ${CONF_FILE}
```

Comment:
We may (instead of this one) introduce a helper script that will take the conf file path from the env variable.

This way can even further simplify the docker-compose YAML - no custom image is needed and the only thing that needs to be provided is the env variable pointing to the custom config.

What do you think, @thewind1984 ?

Reply:
See 18ea8a4